docs/install.md

```diff
@@ -341,10 +341,10 @@ for `kubectl kustomize`. For example:
          path: lvm-parameters-patch.yaml
      EOF
      $ cat >my-pmem-csi-deployment/lvm-parameters-patch.yaml <<EOF
-     # pmem-ns-init is in the init container #0. Append arguments at the end.
+     # pmem-driver is in the init container #0. Append arguments at the end.
```

Comment:
It's "the container #0", not "the *init* container #0".

Reply:
Fixed.